index.html

```diff
@@ -608,6 +614,18 @@ <h2>Invoke idle callback timeout algorithm</h2>
         </li>
       </ol>
     </section>
+    <section>
+      <h2>Is high-priority work pending algorithm</h2>
+      <p>The <dfn>is high-priority work pending algorithm</dfn>:</p>
```

Comment:
This should define arguments. e.g. <dfn>is high-priority work pending algorithm</dfn> given <var>event loop</var>

Reply:
Done